src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReplacementService.cs

```diff
@@ -112,6 +121,8 @@ public class ReplacementService : IReplacementService
                         return ValidationActionResult(ReplacementMustBeValid);
                     }
 
+                    await replacementPlan.DataBlocks.ForEachAsync(plan =>
```

Comment:
It will be good to move away from our home baked extensions to the `System.Linq.Async` extensions but appreciate this is using `ForEachAsync` for consistency without changing everything!